README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
+
+# verify it works
+aws s3 ls
+```
+
+### ECR repository & S3 bucket
+
+The CI pipeline creates two artifacts:
+
+- An action server docker **image**, which is pushed to an **AWS ECR** repository.
+- A trained rasa **model**, which is copied to an **AWS S3** bucket
+
+Run these commands to create the storage for these artifacts. These commands run the AWS CLI and create an ECR repository and an S3 bucket:
+
+```bash
+# create ECR repository with name `financial-demo`
+make aws-ecr-create-repository
+
+# create S3 bucket with name `rasa-financial-demo`
+make aws-s3-create-bucket
+```
+
+
+
+## EKS production cluster
+
+If the production cluster is already set up, you can skip this section.
+
+This section describes the initial deployment of the financial-demo bot on an EKS production cluster. 
+
+This initial deployment is done manually. After that, the deployment is maintained & upgraded automatically by the CI/CD pipeline.
+
+### Preparation
+
+#### Install eksctl
+
+See the [installation instructions](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-eksctl
+```
+
+#### Install kubectl
+
+See the [installation instructions](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-kubectl
+```
+
+#### Install helm
+
+See the [installation instructions](https://helm.sh/docs/intro/install/)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-helm
+```
+
+#### Install jp
+
+See the [installation instructions](https://github.com/jmespath/jp#installing)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-jp
+```
+
+#### Set environment variables
+
+There are many ways to set the required environment variables in your local environment. 
+
+One way is to create a file  `./secret/envs_export`, with this content:
+
+```bash
+# source this file to set the environment variables the same as the GitHub secrets.
+export GCR_AUTH_JSON_PRIVATE_KEY_ID=...
+export GCR_AUTH_JSON_PRIVATE_KEY='-----BEGIN PRIVATE KEY-...-END PRIVATE KEY-----\n'
+export GCR_AUTH_JSON_CLIENT_EMAIL='...'
+export GCR_AUTH_JSON_CLIENT_ID=...
+export GLOBAL_POSTGRESQL_POSTGRESQLPASSWORD=...
+export GLOBAL_REDIS_PASSWORD=...
+export RABBITMQ_RABBITMQ_PASSWORD=...
+export RASAX_INITIALUSER_USERNAME=admin
+export RASAX_INITIALUSER_PASSWORD=...
+export RASAX_JWTSECRET=...
+export RASAX_PASSWORDSALT=...
+export RASAX_TOKEN=...
+export RASA_TOKEN=...
+export AWS_ELASTIC_IP=...
+```
+
+Then, create the environment variables with the command:
+
+```bash
+source ./secret/envs_export
+```
+
+### Create the EKS cluster
+
+Create the EKS production cluster, with name `financial-demo-<current branch name>`:
+
+```bash
+make aws-eks-cluster-create AWS_EKS_CLUSTER_NAME=financial-demo-production
+```
+
+Some other useful 'Makefile functions' to interact with the EKS cluster:
```

Comment:
I think it'd be helpful to create a separate section in the README documenting all the makefile functions and what they do.

Reply:
I was thinking about it, but I didn't do it because it introduces something that might be hard to maintain.
I also wonder how often someone would  read these explanation in the README, instead of just relying on the name of the makefile functions. I tried to make them kind of self explanatory.

Reply:
@b-quachtran 
Ah, yes, thank you!
I had removed the help section while developing the Makefile but forgot to put it back in.

Please check how it looks.